### PR TITLE
docs: add example for: awscc_opensearchserverless_access_policy

### PR DESCRIPTION
**Pull request by troyparrett**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

File: examples/resources/awscc_opensearchserverless_access_policy/access_policy.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
+resource "awscc_opensearchserverless_access_policy" "os" {
+  name = "test-os-security-policy"
```
Comment (wellsiau-aws) [inline examples/resources/awscc_opensearchserverless_access_policy/access_policy.tf:2]: suggest to align the indent with the line below

File: examples/resources/awscc_opensearchserverless_access_policy/access_policy.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      {
+        "ResourceType" = "index",
+        "Resource" = [
+          "index/*/*"
+        ],
+        "Permission" = [
+          "aoss:*"
+        ]
+        }, {
+        "ResourceType" = "collection",
```
Comment (wellsiau-aws) [inline examples/resources/awscc_opensearchserverless_access_policy/access_policy.tf:15]: Let's pretty this up 

```suggestion
      {
        "ResourceType" = "index",
        "Resource" = [
          "index/*/*"
        ],
        "Permission" = [
          "aoss:*"
        ]
      },
      {
        "ResourceType" = "collection",
```

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
minor nit pick, otherwise looks good to me

**Comment by troyparrett** [comment]
@wellsiau-aws please review again

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thanks!

**Pull request merged**